### PR TITLE
blog: February newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Just in time!

### CODE REVIEW

File: _posts/2017-02-28-foreman-community-newsletter---february-2017.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+- community
+- fosdem
+- cfgmgmtcamp
+excerpt: |
+  A bumper roundup all the release, events, and plugin news since Christmas, as
+  well as the conference trip report!
+---
+
+Apologies for the lack of a January newsletter - the conferences in the Czech
+Republic and Belgium kept us all very busy! Here's the roundup of the last few
```
Comment (mmoll) [inline _posts/2017-02-28-foreman-community-newsletter---february-2017.md]: "Czechia" is the official short name now: http://www.independent.co.uk/news/world/europe/czechia-czech-republic-new-name-english-speakers-a7323596.html :trollface:

Comment (GregSutcliffe) [reply _posts/2017-02-28-foreman-community-newsletter---february-2017.md]: Ta! Fixed

File: _posts/2017-02-28-foreman-community-newsletter---february-2017.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+view enhancements and Tracer support, it's a big update. See [the release
+thread][k3_3] for details - and do watch out for the Puppet 3 -> Puppet 4
+upgrade bug (which is scheduled to be fixed in 3.3.1)
+
+[k3_3]: https://groups.google.com/d/msg/foreman-users/u9gKSKMZXZ0/04BU83UqAgAJ
+
+### European Conference Extravaganza
+
+As always in late January & early February, we have 2 weeks of conference
+madness. With that behind us now, here's a round up of what happened at them.
```
Comment (ohadlevy) [inline _posts/2017-02-28-foreman-community-newsletter---february-2017.md:47]: what about some pictures? :)

Comment (GregSutcliffe) [reply _posts/2017-02-28-foreman-community-newsletter---february-2017.md:47]: Yeah, wanted to get the draft up. will get some added in the next hour

**Comment by GregSutcliffe** [comment]
Photos added, anything else? :)

**Comment by ohadlevy** [comment]
thanks @GregSutcliffe

**Pull request merged**